app/models/spree/flow_io_order_decorator.rb

```diff
@@ -23,6 +23,13 @@ def flow_order
       flow_data&.[]('order')
     end
 
+    def flow_order?
```

Comment:
I'd call it more specifically, something like `flow_order_with_payments?`. Because there will be lot of flow orders without payments, but still flow orders.

Reply:
Oh sure! I am ok with that renaming! I will limit this method to only check payments though, since it will make more sense. Thanks Aurel!